readme.txt

```diff
@@ -1,541 +0,0 @@
-/*  _______         ____    __         ___    ___
```

Comment:
Hm. Is it intentional that you delete the whole README? Besides some outdated information it still has some useful information. At least the ASCII-Art should stay IMO 😁

Reply:
No comment on the ascii-art, I don't particularly care. Just felt like it served no purpose and made the title harder to read.

As to deleting the whole readme, what information was useful ?

Reply:
The features mostly. Of course it would need to be updated and some points can be removed or reworded, but I think it is useful to have a readme that explains what the projects does and what its features are. But I understand that @kode54 probably has the best overview on the current features.

Reply:
Ok, I have just done a quick conversion to Markdown and upon closer reading we can probably indeed delete 70-80% of the readme. I do not have the time to finish this today, but I am volunteering to polish the README and I will open a separate PR for discussion/comments in order to not derail the discussion here.

Reply:
See #42 

Maybe you can remove the deletion of the readme from your branch so that our changes do not clash.
Oh, and for the record, I do not have strong opinions on the ASCII art, but I think it does not do any harm either.

Reply:
Yup, let's split this work to multiple PR's. Will make more PR's and close this later.